e2e-tests/graphql-spring-examples/pom.xml

```diff
@@ -0,0 +1,142 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.2.RELEASE</version>
+        <relativePath/>
+    </parent>
```

Comment:
this is bit problematic, as breaking the module hierarchy (which has a few side-effects). Should rather be:
```
<parent>
        <groupId>org.evomaster</groupId>
        <artifactId>evomaster-e2e-tests</artifactId>
        <version>0.2.1-SNAPSHOT</version>
    </parent>
```